inc/options/install.php

```diff
@@ -1,100 +1,422 @@
-        <li><?php _e( '<em>Optional:</em> On the "<a href="admin.php?page=w3tc_dbcache">Database Cache</a>" tab the recommended settings are preset. If using a shared hosting account use the "disk" method with caution; in either of these cases the response time of the disk may not be fast enough, so this option is disabled by default.', 'w3-total-cache' ); ?></li>
-        <li><?php _e( '<em>Optional:</em> On the "<a href="admin.php?page=w3tc_objectcache">Object Cache</a>" tab the recommended settings are preset. If using a shared hosting account use the "disk" method with caution, the response time of the disk may not be fast enough, so this option is disabled by default. Test this option with and without database cache to ensure that it provides a performance increase.', 'w3-total-cache' ); ?></li>
-        <li><?php _e( '<em>Optional:</em> On the "<a href="admin.php?page=w3tc_cachegroups">User Agent Groups</a>" tab, specify any user agents, like mobile phones if a mobile theme is used.', 'w3-total-cache' ); ?></li>
-    </ol>
+	<h3 id="initial"><?php esc_html_e( 'Initial Installation', 'w3-total-cache' ); ?></h3>
+	<ol>
+		<li>
+			<?php esc_html_e( 'Set the permissions of wp-content/ back to 755, e.g.:', 'w3-total-cache' ); ?>
+			<pre class="console"># chmod 755 /var/www/vhosts/domain.com/httpdocs/wp-content/</pre>
+		</li>
+		<li>
+			<?php
+			echo wp_kses(
+				sprintf(
+					// translators: 1 opening HTML a tag to W3TC General admin page, 2 closing HTML a tag.
+					__(
+						'On the "%1$sGeneral%2$s" tab and select your caching methods for page, database and minify. In most cases, "disk enhanced" mode for page cache, "disk" mode for minify and "disk" mode for database caching are "good" settings.',
+						'w3-total-cache'
+					),
+					'<a href="admin.php?page=w3tc_general">',
+					'</a>',
+				),
+				array(
+					'a' => array(
+						'href' => array(),
+					),
+				)
+			);
+			?>
+		</li>
+		<li>
+			<?php
+			echo wp_kses(
+				sprintf(
+					// translators: 1 opening HTML a tag to W3TC PageCache admin page, 2 closing HTML a tag.
+					__(
+						'1. The "Compatibility Mode" option found in the advanced section of the "%1$sPage Cache Settings%2$s" tab will enable functionality that optimizes the interoperablity of caching with WordPress, is disabled by default, but highly recommended. Years of testing in hundreds of thousands of installations have helped us learn how to make caching behave well with WordPress. The tradeoff is that disk enhanced page cache performance under load tests will be decreased by ~20% at scale.',
+						'w3-total-cache'
+					),
+					'<a href="admin.php?page=w3tc_pgcache">',
```

Comment:
Use `esc_url()` and `admin_url()` for the `href`.

Reply:
Added esc_url and admin_url to href